Gemfile.lock

```diff
@@ -140,7 +140,7 @@ GEM
       mime-types
     chef-apply (0.1.18)
       chef (>= 14.0)
-      chef-dk (3.1.6)
```

Comment:
I have no idea how `3.1.6` got in here because it is not in Rubygems yet. Maybe somehow it got pointed to the version I had checked out from master? The `chef-apply` [gem description](https://rubygems.org/gems/chef-apply) on Rubygems has the correct constraint (`>= 3.0`)